packages/docs-sidenav/style.module.css

```diff
@@ -167,6 +167,8 @@
   & code {
     margin-left: 2px;
     font-size: 0.875rem;
+    font-family: var(--font-monospace);
+    line-height: 1.375rem;
```

Comment:
Doing a bit of back-filling for global `code` styles - now these are intentionally set within the component 👍